scm/github/repo.go

```diff
@@ -258,13 +258,20 @@ func (c *client) Update(u *library.User, r *library.Repo, hookID int64) error {
 	}
 
 	// send API call to update the webhook
-	_, _, err := client.Repositories.EditHook(ctx, r.GetOrg(), r.GetName(), hookID, hook)
+	_, resp, err := client.Repositories.EditHook(ctx, r.GetOrg(), r.GetName(), hookID, hook)
+
+	// track if webhook cannot be captured from GitHub (no longer found),
+	// indicating the webhook has been manually deleted from GitHub
+	isWebhookDel := false
+	if resp.StatusCode == http.StatusNotFound {
+		isWebhookDel = true
+	}
 
 	if err != nil {
-		return err
+		return isWebhookDel, err
 	}
 
-	return nil
+	return isWebhookDel, nil
```

Comment:
Thoughts on shuffling this a bit to
```go
	if err != nil {
		return resp.StatusCode == http.StatusNotFound, err
	}

	return true, nil
```

That way the affirmative `true` is when the webhook does exist, and `false` is when the webhook doesn't exist. Or we could return the code itself and let the API do the logic? Not really sure the cleanest way...

Reply:
I did implement the inverse for now, for mvp.